src/lsm/table_memory.zig

```diff
@@ -84,6 +102,8 @@ pub fn TableMemoryType(comptime Table: type) type {
             };
 
             table.* = .{
+                .sorted_runs = [_]?SortedRun{null} ** sorted_runs_max,
```

Comment:
will just

```suggestion
                .sorted_runs = .{null} ** sorted_runs_max,
```

work?

Reply:
Did I understand it correctly from the comment below that we do not make it optional now?

Reply:
Yeah, these are separate suggestions:

* remove optionals in favor of invalid object that will blow up on use anyway
* Simplify array initialization by omitting the type